templates/plugin/python/module_header.conf.erb

```diff
@@ -1,3 +1,4 @@
+<Plugin "python">
```

Comment:
maybe it's now time to convert this to an epp template?

Reply:
I might tackle but best as a separate MR I think.